components/Facility/index.js

```diff
@@ -0,0 +1,64 @@
+import styled from "styled-components";
+import Image from "next/image";
+import Link from "next/link";
+
+export default function Facility({ facility }) {
+  const {
+    id,
+    name,
+    category,
+    adress,
+    target,
+    requirements,
+    image,
+    link,
+    filterCriteria,
+  } = facility;
+
+  return (
+    <>
+      <h3>{name}</h3>
+      <StyledSection>
+        <StyledImage
+          alt={`Logo der Einrichtung ${name}`}
+          src={image}
+          width={120}
+          height={80}
+          priority
+        />
+        <StyledArticle>
+          <p>Kategorie: {category}</p>
+          <p>
+            Adresse:
+            <StyledLink href={link}> {adress}</StyledLink>
+          </p>
+          <p>Zielgruppe: {target}</p>
+          <p>Zugangsvoraussetzungen: {requirements}</p>
+        </StyledArticle>
+      </StyledSection>
+    </>
+  );
+}
+const StyledImage = styled(Image)`
+  object-fit: contain;
+  layout: responsive;
```

Comment:
I'm not sure what `layout` is for? Do you mean a property for the Next Image, like `layout="responsive"`?